scipy/signal/lti_conversion.py

```diff
@@ -334,15 +334,15 @@ def ss2zpk(A, B, C, D, input=0):
 
 def cont2discrete(system, dt, method="zoh", alpha=None):
     """
-    Transform a continuous to a discrete state-space system.
+    Transform a continuous system to a discrete one.
 
     Parameters
     ----------
     system : a tuple describing the system or an instance of `lti`
         The following gives the number of elements in the tuple and
         the interpretation:
 
-            * 1: (instance of `lti`)
+            * 1: (instance of `lti`,) in a tuple or bare instance of `lti`
```

Comment:
the text "tuple describing the system of an instance of `lti`" text above covers this already.  "bare lti" doesn't quite fit here, since we don't have a tuple of length 1.

Reply:
Yes, you are right. I revert this change. I was just a bad attempt to be very clear (because as a user, the first time I came to this documentation, it was not clear for me if I had to place the lti in tuple or not). If we read it carefully, it just claims that both are possible.

Reply:
I agree that it is confusing to support both option. But a function that does not do what its documentation claims is even more confusing for the users. It is better to make the code do what its documentation claims until someone decides what to do with this because it has some package-wide implications (it appears at multiple places across scipy-signal, even if the functions don't always honor the documentation claims).

Reply:
Give me a few days to add tests in the files you mentioned above (I am busy now but will have some time to look at it soon).

Reply:
OK, I added the relevant tests to `scipy/signal/tests/test_cont2discrete.py` in `TestC2dLti().test_c2d_ss()` and `TestC2dLti.test_c2d_tf` and checked that the bare and length-one-tuple cases give the same result as the `.to_discrete` method.

While adding those tests, I detected a mistake in the sampling time of `TestC2dLti.test_c2d_tf` that forced to use a very large tolerance in order to have a passing test. In short, the reference discretized system to target is (seemingly obtained with Matlab by the designer of the test):

```
        num_res = np.array([0.5, -0.485149004980066])
        den_res = np.array([1.0, -0.980198673306755])
```

I do not have Matlab but I checked with Octave that these reference results actually correspond to a sampling time of `dt = 0.05` and not `0.005` as initially set in the test. This bad sampling time lead to significantly different results than the reference, but the test was enforced to pass by using a large tolerance. This is now fixed and a normal tight tolerance can now be used again.

If you agree with the changes, it seems we should be good now.